services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Persistence.kt

```diff
@@ -181,6 +181,34 @@ class HankeEntity(
     var polyHaitta: Haitta13? = null
     var tarinaHaitta: Haitta13? = null
 
+    // --------------- Hankkeen haittojenhallinta ---------
+    var pyoraliikenteenPaareitit: String? = null
+    var merkittavatJoukkoliikennereitit: String? = null
+    var autoliikenteenRuuhkautuminen: String? = null
+    var omanJaMuidenHankkeidenKiertoreitit: String? = null
+    var muutHankkeet: String? = null
+    var moottoriLiikenteenViivytykset: String? = null
```

Comment:
Kaikilla tästä alaspäin on kälikuvissa erillinen namiska "Ei koske hanketta". Onko tätä mallinnettu jotenkin?